Compiler warnings when building wheel (Python 3.8, GCC 10 on Debian)

There are some warnings when building the C extension on Debian stable:

```
$ python setup.py bdist_wheel
[...]
running build_ext
building 'zfec._fec' extension
creating build/temp.linux-x86_64-3.8
creating build/temp.linux-x86_64-3.8/zfec
x86_64-linux-gnu-gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -Izfec/ -I/home/sajith/projects/tahoe/zfec-github/venv/include -I/usr/include/python3.8 -c zfec/fec.c -o build/temp.linux-x86_64-3.8/zfec/fec.o -std=c99
zfec/fec.c: In function ‘fec_new’:
zfec/fec.c:447:38: warning: comparison of integer expressions of different signedness: ‘unsigned int’ and ‘int’ [-Wsign-compare]
  447 |     for (p = tmp_m + k, row = 0; row < n - 1; row++, p += k)
      |                                      ^
x86_64-linux-gnu-gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -Izfec/ -I/home/sajith/projects/tahoe/zfec-github/venv/include -I/usr/include/python3.8 -c zfec/_fecmodule.c -o build/temp.linux-x86_64-3.8/zfec/_fecmodule.o -std=c99
zfec/_fecmodule.c: In function ‘Encoder_encode’:
zfec/_fecmodule.c:137:19: warning: comparison of integer expressions of different signedness: ‘size_t’ {aka ‘long unsigned int’} and ‘int’ [-Wsign-compare]
  137 |     for (i = 0; i < self->mm - self->kk; i++)
      |                   ^
[...]
```

Python and compiler versions:

```
$ python --version
Python 3.8.5

$ x86_64-linux-gnu-gcc --version
x86_64-linux-gnu-gcc (Debian 10.1.0-6) 10.1.0
Copyright (C) 2020 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

--- a/zfec/_fecmodule.c
+++ b/zfec/_fecmodule.c
@@ -134,7 +134,7 @@ Encoder_encode(Encoder *self, PyObject *args) {
     if (!PyArg_ParseTuple(args, "O|O:Encoder.encode", &inblocks, &desired_blocks_nums))
         return NULL;
 
-    for (i = 0; i < self->mm - self->kk; i++)
+    for (i = 0; i < (size_t)self->mm - (size_t)self->kk; i++)
         pystrs_produced[i] = NULL;
 
     if (desired_blocks_nums) {

--- a/zfec/fec.c
+++ b/zfec/fec.c
@@ -444,7 +444,7 @@ fec_new(unsigned short k, unsigned short n) {
     tmp_m[0] = 1;
     for (col = 1; col < k; col++)
         tmp_m[col] = 0;
-    for (p = tmp_m + k, row = 0; row < n - 1; row++, p += k)
+    for (p = tmp_m + k, row = 0; row + 1 < n; row++, p += k)
         for (col = 0; col < k; col++)
             p[col] = gf_exp[modnn (row * col)];
 

